client/layout/masterbar/style.scss

```diff
@@ -1401,3 +1414,53 @@ a.masterbar__quick-language-switcher {
 		opacity: 1;
 	}
 }
+
+.masterbar__item.has-global-border .item-border-wrapper {
```

Comment:
All of this is what allows us to position and shape elements for that awkward outward border radius/curve on the bottom of the selected global items. Maybe there is a less complex way I am unaware of, but simply leveraging pseudo elements on existing items didn't seem to give us enough control.